psp/models/recent_history.py

```diff
@@ -136,6 +138,9 @@ def __init__(
         num_days_history: int = 7,
         nwp_dropout: float = 0.1,
         nwp_tolerance: Optional[float] = None,
+        satellite_dropout: float = 0.1,
```

Comment:
as using "sat" elsewhere, such as sat_data_sources. Maybe would be better for sat_tolerance instead of satellite_tolerance?

Reply:
II think i like it full, versbose, but ill make it consitent